### PR TITLE
Add tool tips to hard-coded UI elements

### PR DESCRIPTION
**Pull request by janper**
The tool tips (temporarily) replace the missing help and user manual.
In the future, there can be an UI option to turn the tool tips off.

### CODE REVIEW

File: src/ui.rs [hunk 1]
```diff
@@ -520,16 +606,50 @@ impl<'a> UiFrame<'a> {
...
-                if ui.button(imgui::im_str!("Open project"), [-f32::MIN_POSITIVE, 0.0]) {
+                if ui.is_item_hovered() {
+                    ui.tooltip(|| {
+                        let wrap_token = ui.push_text_wrap_pos(WRAP_POS_TOOLTIP_TEXT_PIXELS);
+                        ui.text("SAVE PROJECT INTO A .hurban FILE\n\
+                        \n\
+                        Saves the current project into a .hurban file. \
+                        When used for the first time, opens a system dialog to specify save file location.\n\
+                        \n\
+                        The .hurban project file contain only the operation sequence. It does not contain any \
```
Comment (yanchith) [inline src/ui.rs]: contains

File: src/ui.rs [hunk 2]
```diff
@@ -520,16 +606,50 @@ impl<'a> UiFrame<'a> {
...
 
+                if ui.is_item_hovered() {
+                    ui.tooltip(|| {
+                        let wrap_token = ui.push_text_wrap_pos(WRAP_POS_TOOLTIP_TEXT_PIXELS);
+                        ui.text("OPEN PROJECT FROM A .hurban FILE\n\
+                        \n\
+                        Opens the sequence of operations saved in a .hurban file. \
+                        The current unsaved project will be lost.\n\
+                        \n\
+                        The .hurban project file contain only the operation sequence. It does not contain any \
```
Comment (yanchith) [inline src/ui.rs]: contains

File: src/ui.rs [hunk 3]
```diff
@@ -568,6 +688,41 @@ impl<'a> UiFrame<'a> {
...
             .position([MARGIN, MARGIN], imgui::Condition::Always)
             .build(ui, || {
+                if ui.is_item_hovered() {
+                    ui.tooltip(|| {
+                        let wrap_token = ui.push_text_wrap_pos(WRAP_POS_TOOLTIP_TEXT_PIXELS);
+                        ui.text(
+                            "SEQUENCE OF OPERATIONS\n\
+                            \n\
+                            An ordered list of operations that generate the viewport geometry. \
+                            Then the sequence is run, the operations are being executed one after \
```
Comment (yanchith) [inline src/ui.rs]: Then -> When

File: src/ui.rs [hunk 4]
```diff
@@ -568,6 +688,41 @@ impl<'a> UiFrame<'a> {
...
+                            This is useful when some parameters need to be adjusted only after the results \
+                            of subsequent operations can be visually evaluated. This approach is a gateway \
+                            to full-fledged parametric modeling paradigm.\n\
+                            \n\
+                            The .hurban project files can be build as a tool for achieving certain manipulations \
+                            with mesh geometry and because they don't contain the imported mesh models, \
+                            one sequence of operations (one project file) can be reused for various input mesh \
+                            models. Hence, H.U.R.B.A.N. selector is not only a geometry transformation tool, \
+                            but also a tool-building platform and the project files ara not only geometries but \
+                            also tools.",
```
Comment (yanchith) [inline src/ui.rs:731]: :heart: I need your drugs

**Review by yanchith — COMMENTED** [review]
Nice, although I am starting to be worried about keeping all of these docs up to date with what the software does. Also, as you say, we will probably need some option to tune the amount of tooltips down later.

Found a couple of typos/

File: src/ui.rs (hunk 4 above)
Comment (janper) [reply src/ui.rs:731]: it's called endorphin and dopamine

**Comment by janper** [comment]
Do you want me to implement a "mute" button for tool tips? To be honest, I think everyone will need some help using this. Ever.

**Comment by yanchith** [comment]
> Do you want me to implement a "mute" button for tool tips?

Not now for sure. Later it might be useful for power users... like you or me :smile:

**Comment by janper** [comment]
> > Do you want me to implement a "mute" button for tool tips?
> 
> Not now for sure. Later it might be useful for power users... like you or me 😄

and @ondrowan !

**Review by yanchith — APPROVED** [review]

**Pull request merged**